packages/timeline-state-resolver-types/src/telemetrics.ts

```diff
@@ -0,0 +1,14 @@
+import { DeviceType, TSRTimelineObjBase } from './index'
+
```

Comment:
This file should export a `MappingTelemetrics` interface.
```suggestion
export interface MappingTelemetrics extends Mapping {
	device: DeviceType.TELEMETRICS
}
```

Reply:
I must admit I don't see the value of making the `MappingTelemetrics` since it doesn't add any new attributes to `Mapping`. 

All it does is that it ensures that I can't write any other value than `DeviceType.TELEMETRICS` when creating my mappings. (I still have to actually write `DeviceType.TELEMETRICS` when creating my mappings, so it doesn't even save me that trouble...)
But there is nothing stopping me from still just using `Mapping` when creating my mappings.

So it ensures that the developer use the correct DeviceType, but before that the developer allready needs to know what DeviceType he needs in order to select the correct interface. Aka, we try to help the developer to select the correct thing, but in order for it to help the developer select the correct thing, the developer needs to allready have selected the correct thing, so this becomes redundant.

Now, if `MappingTelemetrics` also included new attributes, then this is a completely different matter and then I would see the value of creating it.

Reply:
It's to provide some extra level of type safety for the consumers of the library. If at some point in the future we decide that the Telemetrics integration mappings requires some extra parameters, it would be immediately spotted after upgrading the package by those who used `MappingTelemetrics`. ([a real-world example](https://github.com/SuperFlyTV/SuperConductor/blob/HEAD/apps/app/src/lib/TSRMappings.ts#L351-L364))